packages/design-system-docs/src/pages/components/Review/_Review.docs.scss

```diff
@@ -14,9 +14,9 @@ Style guide: patterns.review
 
 The `Review` component wraps up the styles and markup required to make a single reviewable row. They are designed to be used either alone, as a single row, or multiple rows can be combined together one after the other to present more information. The component can take text or HTML as row content.
 
-@react-example Review.example.jsx
+@react-example Review.example.tsx
```

Comment:
I don't think we've been converting our example files to Typescript. Not sure why though.